eo-maven-plugin/src/test/java/org/eolang/maven/OyFallbackTest.java

```diff
@@ -53,10 +54,14 @@ void putsObjectToLocalCache(@TempDir final Path path) throws Exception {
             new OyCaching(
                 branch,
                 path,
-                name -> {
-                    counter.incrementAndGet();
-                    return new InputOf("[] > main\n");
-                }
+                new OyLambda(
+                    new UncheckedFunc<>(
```

Comment:
@MikhailLipanin Consider to add one more constructor for `OyLambda` in order to allow it to accept simple lambda.

Reply:
done